common/src/TrenchBroomApp.cpp

```diff
@@ -105,8 +105,8 @@ namespace TrenchBroom {
             setApplicationName("TrenchBroom");
             // Needs to be "" otherwise Qt adds this to the paths returned by QStandardPaths
             // which would cause preferences to move from where they were with wx
-            setOrganizationName("");
-            setOrganizationDomain("com.kristianduske");
+            setOrganizationName(""); // FIXME: this should be "TrenchBroom", regardless of the preferences moving
```

Comment:
AFAIK there's no urgent reason to give an organization name (see also krita does the same: https://github.com/KDE/krita/blob/2e501442362ab45263cad8351c9d290e9086d77f/libs/ui/KisApplication.cpp#L151 )
whereas I consider moving our config directories like API breakage which we need to avoid if possible, so I disagree with the comment

Reply:
alright, removed the comment and left the vendor name blank